lib/utils/HDFDatabase.h

```diff
@@ -225,21 +263,132 @@ class HDFDatabase : public Database
         int nelements,
         int offset,
         int block_size,
-        int stride);
+        int stride,
+        const bool distributed=false);
 
-private:
-    /**
-     * @brief Unimplemented copy constructor.
-     */
-    HDFDatabase(
-        const HDFDatabase& other);
+    // /**
+    //  * @brief Creates a new HDF5 database file with the supplied name,
+    //  *        extending it with 6 digits indicating the process rank.
+    //  *
+    //  * @param[in] file_name Name of HDF5 database file to create.
+    //  * @param[in] comm      MPI communicator to obtain the process rank.
+    //  *
+    //  * @return True if file create was successful.
+    //  */
+    // virtual bool
+    // create_parallel(
+    //     const std::string& file_name,
+    //     const MPI_Comm comm) override
+    // {
+    //     CAROM_VERIFY(!file_name.empty());
+    //     MPI_Comm_rank(comm, &d_rank);
 
-    /**
-     * @brief Unimplemented assignment operator.
-     */
-    HDFDatabase&
-    operator = (
-        const HDFDatabase& rhs);
+    //     std::string ext_filename(file_name);
+    //     char tmp[10];
+    //     sprintf(tmp, ".%06d", d_rank);
+    //     ext_filename += tmp;
+
+    //     return create(ext_filename);
+    // }
+
+    // /**
+    //  * @brief Opens an existing HDF5 database file with the supplied name,
+    //  *        extending it with 6 digits indicating the process rank.
+    //  *
+    //  * @param[in] file_name Name of existing HDF5 database file to open.
+    //  * @param[in] type      Read/write type ("r"/"wr")
+    //  * @param[in] comm      MPI communicator to obtain the process rank.
+    //  *
+    //  * @return True if file open was successful.
+    //  */
+    // virtual
```

Comment:
There are a few places with some commented out blocks of code. Are those needed for something in the future, or can they be removed?